{"title": "logging_exceptions", "access": "public", "summary": "Self-logging exceptions: Attach log messages to exceptions and output them conditionally.", "updated": "2018-04-30", "detail": {"meta": {"license": "MIT", "license_url": "http://opensource.org/search/node/MIT", "home": "https://github.com/Bernhard10/logging_exceptions", "home_url": "https://github.com/Bernhard10/logging_exceptions", "documentation": null, "documentation_url": null, "development": "https://github.com/Bernhard10/logging_exceptions", "development_url": "https://github.com/Bernhard10/logging_exceptions", "download_count": 3571, "last_upload": "Last upload: 1 year and 11 months ago"}, "installers": ["conda install -c conda-forge logging_exceptions", "conda install -c conda-forge/label/gcc7 logging_exceptions", "conda install -c conda-forge/label/cf201901 logging_exceptions", "conda install -c conda-forge/label/cf202003 logging_exceptions"], "note": " This package contains files in non-standard\n           labels", "platform_versions": [["\n  ", "v0.1.8"]], "description": "<div class=\"panel\">\n      <h3>Description</h3>\n      \n        \n        \n        \n      \n      \n    </div>", "files": [{"type": "conda", "size": "8.3 kB", "file": {"name": "noarch/logging_exceptions-0.1.8-py_0.tar.bz2", "href": "https://anaconda.org/conda-forge/logging_exceptions/0.1.8/download/noarch/logging_exceptions-0.1.8-py_0.tar.bz2"}, "uploaded": null, "uploader": "conda-forge", "downloads": 1906, "labels": [{"name": "main", "label": "https://anaconda.org/conda-forge/repo/files?type=any&label=main"}, {"name": "cf202003", "label": "https://anaconda.org/conda-forge/repo/files?type=any&label=cf202003"}, {"name": "gcc7", "label": "https://anaconda.org/conda-forge/repo/files?type=any&label=gcc7"}, {"name": "cf201901", "label": "https://anaconda.org/conda-forge/repo/files?type=any&label=cf201901"}]}, {"type": "conda", "size": "8.2 kB", "file": {"name": "noarch/logging_exceptions-0.1.6-py_0.tar.bz2", "href": "https://anaconda.org/conda-forge/logging_exceptions/0.1.6/download/noarch/logging_exceptions-0.1.6-py_0.tar.bz2"}, "uploaded": null, "uploader": "conda-forge", "downloads": 1665, "labels": [{"name": "main", "label": "https://anaconda.org/conda-forge/repo/files?type=any&label=main"}, {"name": "cf202003", "label": "https://anaconda.org/conda-forge/repo/files?type=any&label=cf202003"}, {"name": "cf201901", "label": "https://anaconda.org/conda-forge/repo/files?type=any&label=cf201901"}]}], "labels": [{"name": "", "label": "0.1.8"}, {"name": "", "label": "0.1.8"}, {"name": "", "label": "0.1.8"}, {"name": "", "label": "0.1.8"}], "badges": ["https://anaconda.org/conda-forge/logging_exceptions/badges/version.svg", "https://anaconda.org/conda-forge/logging_exceptions/badges/latest_release_date.svg", "https://anaconda.org/conda-forge/logging_exceptions/badges/latest_release_relative_date.svg", "https://anaconda.org/conda-forge/logging_exceptions/badges/platforms.svg", "https://anaconda.org/conda-forge/logging_exceptions/badges/license.svg", "https://anaconda.org/conda-forge/logging_exceptions/badges/downloads.svg", "https://anaconda.org/conda-forge/logging_exceptions/badges/installer/conda.svg"]}}